Form/Type/Embedded/OptionType.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Can this OptionType be deleted? Because I can't find its use anywhere in this Bundle.